src/app/clusters/group-key-mgmt-server/group-key-mgmt-server.cpp

```diff
@@ -639,7 +639,7 @@ bool emberAfGroupKeyManagementClusterKeySetRemoveCallback(
     CHIP_ERROR err          = provider->RemoveKeySet(fabricIndex, commandData.groupKeySetID);
 
     Status status = Status::Success;
-    if (CHIP_ERROR_KEY_NOT_FOUND == err)
+    if (CHIP_ERROR_NOT_FOUND == err)
```

Comment:
It appears that maybe the issue is that some GroupDataProvider API calls reutrn CHIP_ERROR_NOT_FOUND and others CHIP_ERROR_KEY_NOT_FOUND.

Reply:
ah. OK, I'll check both on the other side